swagger_server/models/base_model_.py

```diff
@@ -40,9 +40,11 @@ def to_dict(self):
             elif isinstance(value, dict):
                 result[attr] = dict(
                     map(
-                        lambda item: (item[0], item[1].to_dict())
-                        if hasattr(item[1], "to_dict")
-                        else item,
+                        lambda item: (
+                            (item[0], item[1].to_dict())
+                            if hasattr(item[1], "to_dict")
+                            else item
+                        ),
```

Comment:
heads-up: this was just a change to make black happy.. :) the pipeline checks were failing because of this formatting issue.

Reply:
This is fixed in main branch. You can merge main to this PR branch if you want to avoid the irrelevant noise. :-)

Reply:
that is true, @sajith! I've reverted this change. I'm glad you bring this up. I didnt catch this (not sure why GitHub didnt raised one alert here)